src/Interpreters/InterpreterSelectWithUnionQuery.h

```diff
@@ -47,7 +47,7 @@ class InterpreterSelectWithUnionQuery : public IInterpreterUnionOrSelectQuery
         const ASTPtr & query_ptr_,
         ContextPtr context_,
         bool is_subquery,
-        Streaming::DataStreamSemanticEx * output_data_stream_semantic = nullptr);
+        Streaming::GetSampleBlockContext * get_sample_block_ctx = nullptr);
```

Comment:
No need to change the signature of this function, is 'is_streaming_output' has been put into DataStreamSemanticEx